notes.md

```diff
@@ -0,0 +1,1900 @@
+# PyTorch 2.0 Release notes
+
+# Highlights
+
+- &lt;Summary to be provided>
+
+# Backwards Incompatible changes
+
+### **Set min supported Python version to 3.8 (#93155)**
+
+Previously the minimum supported version of Python for PyTorch was 3.7. This PR updates the minimum version to require 3.8 in order to install PyTorch. See [Hardware / Software Support ](https://github.com/pytorch/pytorch/blob/893aa5df3f2a475c91ea8eadb1353812e52fb227/RELEASE.md#python) for more information.
+
+### **Drop support for CUDA 10 (#89582)**
+
+This PR updates the minimum CUDA version to 11.0. See the [getting-started](https://pytorch.org/get-started/locally/) for installation or [building from source](https://github.com/pytorch/pytorch#from-source) for more information.
+
+### **Update `torch.tensor` and `nn.Parameter` to serialize all their attributes (#88913)**
+
+Any attribute stored on `torch.tensor` and `torch.nn.Parameter` will be serialized. This aligns the serialization behavior of `torch.nn.Parameter`, `torch.Tensor` and other tensor subclasses
```

Comment:
nit: will be serialized -> will now be seralized.

Reply:
"This aligns the serialization behavior of `torch.nn.Parameter`, `torch.Tensor` and other tensor subclasses"

I actually don't understand this point, in the 1.13 example it looks consistent.  Is the odd person out other tensor subclasses?  Can you add it to the example?

Reply:
@albanD I think I wrote that line based off of this comment: https://github.com/pytorch/pytorch/issues/72129#issuecomment-1079112477

Do any subclasses come to mind as a good example?